src/main/resources/assessment-config/accommodation/accommodation-assessment-v3.yml

```diff
@@ -117,6 +117,11 @@ pages:
           - id: "NO_SUPPORT_NEEDS"
             displayText: "No support needs identified"
             exclusive: true
+      - id: "SUPPORT_REQUIREMENTS_ADDITIONAL_DETAILS"
+        title: "Additional details (optional)"
```

Comment:
I don't think we want the `(optional)` in here as we have a ticket for the UI to add it based on the validation type.